src/openrct2/ride/RideData.h

```diff
@@ -333,6 +340,7 @@ constexpr const RideTypeDescriptor DummyRTD =
     SET_FIELD(PhotoItem, SHOP_ITEM_PHOTO),
     SET_FIELD(BonusValue, 0),
     SET_FIELD(ColourPresets, DEFAULT_FLAT_RIDE_COLOUR_PRESET),
+    SET_FIELD(ColourPreview, {0, 0 }),
```

Comment:
```suggestion
    SET_FIELD(ColourPreview, { SPR_NONE, SPR_NONE }),
```

Reply:
@Gymnasiast putting SPR_NONE does not work straight forward since it is equal to -1 and it's expecting a uint32_t.